scenes/treehouse.scn

```diff
@@ -171,7 +171,7 @@
         0,
         1
       ],
-      "start_url": "https://webaverse.github.io/npc/"
+      "start_url": "https://bru345.github.io/npc/"

```

Comment:
https://github.com/webaverse/npc/pull/1 was merged. Can we switch this back to not be a fork?